addons/account/models/account_move.py

```diff
@@ -1708,11 +1708,26 @@ def _check_balanced(self):
         if not moves:
             return
 
+        postcommit_check_balanced = self.env.context.get('postcommit_check_balanced')
+        if postcommit_check_balanced == []:
```

Comment:
Is this only activated when `account_accountant` is installed then?

Reply:
Yes

Reply:
See https://github.com/odoo/enterprise/pull/19711

Reply:
I know, that's why I mentioned that module. But it is really weird to have this put like that. Why do we only have this optimization when that module is installed?

Reply:
Because it solves opw ticket and I don't know where else we may need optimization. You may point where else it could be used